products/knownprojects/sql/dcp_application.sql

```diff
@@ -17,49 +17,15 @@ DROP TABLE IF EXISTS _dcp_application;
 WITH
 zap_translated AS (
     SELECT
-        dcp_name,
-        dcp_projectname,
-        dcp_projectbrief,
-        dcp_projectdescription,
-        (CASE
```

Comment:
Curious, we don't need to do this data standardization anymore?

Reply:
nope! this is one of many columns that have always been standardized [during the ZAP export](https://github.com/NYCPlanning/data-engineering/blob/fix-kpdb-build/products/zap-opendata/src/visible_projects.py#L12)

since we now archive the standardized (aka recoded) ZAP export rather than the raw CRM export, that's what KPDB uses

Reply:
My guess would be that it's moved upstream, to the zap data processing? @damonmcc ?